client/blocks/signup-form/index.jsx

```diff
@@ -810,18 +810,14 @@ class SignupForm extends Component {
 	footerLink() {
 		const { flowName, translate } = this.props;
 
-		if ( this.props.positionInFlow !== 0 ) {
-			return;
-		}
-
 		const logInUrl = config.isEnabled( 'login/native-login-links' )
 			? this.getLoginLink()
 			: localizeUrl( config( 'login_url' ), this.props.locale );
 
 		return (
 			<LoggedOutFormLinks>
 				<LoggedOutFormLinkItem href={ logInUrl }>
-					{ flowName === 'onboarding'
+					{ flowName === 'onboarding' || flowName === 'onboarding-user-last'
```

Comment:
IMO the "Already have a WordPress.com account?" copy makes more sense to me when the user step is at the end.

When it says "Log in to create _a_ site for your existing account" I have a sinking feeling that it's going to forget my progress and throw out all my good work. I think "Log in to create _this_ site _with_ your existing account" is more accurate, but the generic copy works too I think.

Reply:
Good point! For the purposes of the A/B test, I can just remove my change here and it'll work fine.

Reply:
Concur. At least to me, the question variant is less of a mouthful. Fewer nouns, less to think about.

Reply:
I've updated this one.